src/components/App/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from './App.module.scss';
+import Header from '../Header';
+import { Router, Route, Switch } from 'react-router-dom';
+import history from '../../history';
+
+import Search from '../../containers/Search';
+import Results from '../../containers/Results';
+import Experiment from '../../containers/Experiment';
+
+const App = () => {
+  return (
+    <div className={styles.App}>
+      <Router history={history}>
```

Comment:
I hope it's not a problem, I went ahead and checked this PR. Looks good! Just one comment here

It may be a good idea to have the router in a separate file, this will make it easier if you want to change the routes later, instead of having the routes mixed with other components of the application. Like `Header` in this case.

I have used nested routes with a `Layout` component that takes care of rendering `Header` where it should be, and leaves space for its children. Something like this for example:

```jsx
const Layout = ({children}) => (
  <div className="layout">
    <div className="layout__header">
      <Header />
    </div>

    <div className="layout__content">
      {children}
    </div>
  </div>
);
```

This would allow you to have the logic concerning the layout of the app in this component, note that if you needed classes like `layout`, `layout__header` or `layout__content` to position the elements they would have to be mixed with the Routes in the current approach.

Then the Router could be something like this:

```jsx
const RouterComponent = () => (
  <Router history={history}>
    <Route path="" component={Layout}>
      <Switch>
        <Route exact path="/" component={Search} />
        <Route path="/results" component={Results} />
        <Route path="/experiment" component={Experiment} />
      </Switch>
    </Route>
  </Router>
);
```

And `App` would only have to render `<RouterComponent />`.

I'm sure there are probably other ways to achieve the same, but the important part I think is to separate the Router logic from the application. Hope this helps :)

Reply:
Yeah, I agree, having the router in a separate component will be a good way to go in the future. But for now, a lot of these routes and components are currently just general placeholders until we get finalized designs for the app, so that'll be better suited for a future PR.